src/nc_util.c

```diff
@@ -22,6 +22,9 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <netdb.h>
+#ifdef SO_REUSEPORT
+#include <linux/version.h>
+#endif
```

Comment:
```suggestion
```

I meant remove the change to includes entirely.

This will also fail to build on non-linux platforms, which is why there was originally the `#ifdef __linux` check in a previous revision of your PR

Reply:
Roger, fixed.